inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/PdfAnnotationEditor.java

```diff
@@ -338,9 +343,17 @@ public void initialize(AjaxRequestTarget aTarget)
         File pdfFile = documentService.getSourceDocumentFile(getModel().getObject().getDocument());
 
         try {
+            File file = new File(getClass().getClassLoader()
```

Comment:
Fetching the file for a resource won't work when the application is packaged as a JAR - it works only when running in the IDE. You need to open the classpath resource as a stream and read it. I'd also suggest to load the substitution table directly from the same location the PdfReader uses such that we don't have to package the resource twice (potentially getting out of sync): `/de/tudarmstadt/ukp/dkpro/core/io/pdf/substitutionTable.xml`

Reply:
```
String substitutionTable = "/de/tudarmstadt/ukp/dkpro/core/io/pdf/substitutionTable.xml";
try (InputStream is = getClass().getClassLoader().getResourceAsStream(substitutionTable)) {
```
InputStream is null, how can I get this right?
Or do I need to have to place the file somewhere?
Thought it would be included with the dependency.
I am not very familiar with this stuff.

Reply:
I now used `ResourceUtils.resolveLocation` from dkpro, seems to work

Reply:
@reckart is it ok now?